gramps/plugins/db/bsddb/read.py

```diff
@@ -1632,6 +1635,23 @@ def get_raw_note_data(self, handle):
     def get_raw_tag_data(self, handle):
         return self.__get_raw_data(self.tag_map, handle)
 
+    def has_handle(self, obj_key, handle):
+        """
+        Determine in a valid handle exists for the obj_key type
```

Comment:
Do you mean   "Determine ~~in~~**if** a valid handle exists for the obj_key type"?